rfc/0084-app-service-level-telemetry.md

```diff
@@ -0,0 +1,532 @@
+  | `sdk_{service}_r_atimedout` | Ambiguous timeout |
+  | `sdk_{service}_r_canceled` | Canceled |
+  | `sdk_{service}_r_total` | Total number of operations |
+
+* `{agent="couchbase-net-sdk/2.4.5.0 (clr/4.0.30319.42000) (os/Microsoft Windows NT 10.0.16299.0)", id="66388CF5BFCF7522/18CC8791579B567C", node="node1", bucket="travel-sample"}` is a list of the tags.
+
+  | Label | Description |
+  | ---- | ---- |
+  | `agent` | agent string of the SDK, as it is used everywhere (HELLO message, logs etc.) |
+  | `id` | SDK identifier string as specified by [SDK-RFC #35](0035-rto.md#client--connection-ids) in section "Client / Connection IDs" |
+  | `node` | The hostname of the node as seen in configuration, and used by the SDK to establish connection. |
+  | `alt_node` | If alternative address is used, this field should be present |
+  | `bucket` | Name of the bucket (if present) |
+
+* `1` is a metric value. Each metric value increment represents a single
+  network operation (e.g. `GET_ALL_REPLICAS` should be counted as `1 +
+  number_of_replicas`).
+* `1695747260` is a metric timestamp, that represents time in milliseconds from
+   the Epoch (1970-01-01 00:00:00 UTC) to the moment of the generation this
+   response.
+
+#### Histograms
+
+    sdk_kv_retrieval_duration_seconds_bucket{le="0.001",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 24054
+    sdk_kv_retrieval_duration_seconds_bucket{le="0.010",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 33444
+    sdk_kv_retrieval_duration_seconds_bucket{le="0.100",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 100392
+    sdk_kv_retrieval_duration_seconds_bucket{le="0.5",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 129389
+    sdk_kv_retrieval_duration_seconds_bucket{le="1",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 133988
+    sdk_kv_retrieval_duration_seconds_bucket{le="2.5",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 139823
+    sdk_kv_retrieval_duration_seconds_bucket{le="+Inf",agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 144320
+    sdk_kv_retrieval_duration_seconds_sum{agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 53423
+    sdk_kv_retrieval_duration_seconds_count{agent="sdk/2.4.5.0",id="66388C/18CC8",bucket="travel-sample",node="node1"} 144320
+
+Where
+
+* `24054` is a number of the points in the bucket
+* `sdk_kv_retrieval_duration_seconds_bucket` is a name of the histogram, where
+  `_bucket` is a special suffix, that signifies that the line should be
+  interpreted as a bucket of the histogram, and
+  `sdk_{service}_retrieval_duration_seconds` prefix describes particular metric
+  of the service. There are two special metrics associated with each histogram:
+  `_sum` and `_count` that represent the overall sum of all buckets, and total
```

Comment:
Wouldn't the value for the `+Inf` bucket always be the same with the `_count` metric?